tools/wptrunner/wptrunner/browsers/chrome.py

```diff
@@ -56,15 +56,29 @@ def executor_kwargs(test_type, server_config, cache_manager, run_info_data,
     if test_type == "testharness":
         capabilities["pageLoadStrategy"] = "none"
 
-    for (kwarg, capability) in [("binary", "binary"), ("binary_args", "args")]:
-        if kwargs[kwarg] is not None:
-            capabilities["goog:chromeOptions"][capability] = kwargs[kwarg]
-
-    if kwargs["headless"]:
-        if "args" not in capabilities["goog:chromeOptions"]:
-            capabilities["goog:chromeOptions"]["args"] = []
-        if "--headless" not in capabilities["goog:chromeOptions"]["args"]:
-            capabilities["goog:chromeOptions"]["args"].append("--headless")
+    chrome_options = capabilities["goog:chromeOptions"]
+    if kwargs["binary"] is not None:
+        chrome_options["binary"] = kwargs["binary"]
+
+    # Here we set a few Chrome flags that are always passed.
+    chrome_options["args"] = []
+    # Allow audio autoplay without a user gesture.
+    chrome_options["args"].append("--autoplay-policy=no-user-gesture-required")
+    # Allow WebRTC tests to call getUserMedia.
+    chrome_options["args"].append("--use-fake-ui-for-media-stream")
+    chrome_options["args"].append("--use-fake-device-for-media-stream")
+    # Shorten delay for Reporting <https://w3c.github.io/reporting/>.
+    chrome_options["args"].append("--short-reporting-delay")
+    # Point all .test domains to localhost for Chrome
+    chrome_options["args"].append("--host-resolver-rules=MAP nonexistent.*.test ~NOTFOUND, MAP *.test 127.0.0.1")
```

Comment:
@LukeZielinski @jgraham I'd rather we didn't hardcode `*.test` here, given we build up the right set of hosts dynamically for every other browser (from `self.config`), but I don't think it's a big deal.